returnn/engine/base.py

```diff
@@ -72,19 +73,11 @@ def get_epoch_model(cls, config):
       start_epoch = int(start_epoch_mode)
       assert start_epoch >= 1
 
-    load_model_epoch_filename = config.value('load', '')
-    if load_model_epoch_filename.endswith(".meta"):
-      load_model_epoch_filename = load_model_epoch_filename[:-len(".meta")]
-    elif load_model_epoch_filename.endswith(".index"):
-      load_model_epoch_filename = load_model_epoch_filename[:-len(".index")]
+    load_model_epoch_filename = get_checkpoint_filepattern(config.value('load', ''))
     if load_model_epoch_filename:
       assert os.path.exists(load_model_epoch_filename + get_model_filename_postfix())
 
-    import_model_train_epoch1 = config.value('import_model_train_epoch1', '')
-    if import_model_train_epoch1.endswith(".meta"):
-      import_model_train_epoch1 = import_model_train_epoch1[:-len(".meta")]
-    elif import_model_train_epoch1.endswith(".index"):
-      import_model_train_epoch1 = import_model_train_epoch1[:-len(".index")]
+    import_model_train_epoch1 = get_checkpoint_filepattern(config.value('import_model_train_epoch1', ''))
```

Comment:
This code here is intended to work in all cases, also when TF is not available. So no TF import should be done here (also not indirectly). So you cannot use things from `tf_util`.